resources/user-agents/browsers/opera-touch/opera-touch-1-7-on.json

```diff
@@ -0,0 +1,99 @@
+{
+  "division": "Opera Touch #MAJORVER#.#MINORVER#",
+  "versions": ["1.10", "1.9", "1.8", "1.7"],
+  "sortIndex": 1045,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Opera Touch #MAJORVER#.#MINORVER# for iOS",
+      "browser": "opera touch",
+      "platform": "iOS",
+      "engine": "WebKit",
+      "device": "general Mobile Device (Apple)",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Opera Touch #MAJORVER#.#MINORVER#",
+        "Version": "#MAJORVER#.#MINORVER#"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#DEVICE##PLATFORM#)*applewebkit*(*khtml*like*gecko*) OPT/#MAJORVER#.#MINORVER#* Mobile/*",
+          "devices": {
+            "iPhone": "iPhone"
+          },
+          "platforms": [
+            "iOS_A_11_4",
```

Comment:
Could probably add iOS 11.0 - 12.0 here (Apple's entry for this says iOS 11.0+: https://itunes.apple.com/us/app/opera-touch/id1411869974?mt=8), here's a UA from my phone on iOS 12:

```
Mozilla/5.0 (iPhone; CPU iPhone OS 12_0 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) OPT/69 Mobile/16A366
```

Also, I noticed that the versions in this file don't represent the versions in the UA at all (for iOS). Should we just add these to the generic file and not add them at all in the versioned file?  Seems like a waste of patterns that won't ever get matched.

Reply:
I found this UA [here](https://github.com/matomo-org/device-detector/pull/5828/files): `Mozilla/5.0 (iPhone; CPU iPhone OS 11_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) OPT/1.10.36 Mobile/15G77`

Reply:
Interesting, I wonder if it's been spoofed...  or is perhaps an unreleased version.

Here's the version history for that app on the apple itunes store:

![screen shot 2018-10-08 at 12 31 58 pm](https://user-images.githubusercontent.com/136907/46626982-8aa50100-cb28-11e8-95b9-95321e9690e3.png)

Reply:
iOS rules are changed